src/frontend/components/PublicVideoLiveJitsi/index.spec.tsx

```diff
@@ -123,20 +124,21 @@ describe('<PublicVideoLiveJitsi />', () => {
 
     expect(mockJitsi).toHaveBeenCalled();
 
-    //  expect chat input to be present (because live panel will contain at least the chat)
-    //  but not visible (because live panel state is configured by default not to show the panel)
-    //  for now, live panel is not unmounted when we don't want to display it because of chat issues
-    expect(screen.getByText('Join the chat')).not.toBeVisible();
+    //  expect chat input to be present
+    expect(screen.getByText('Join the chat')).toBeInTheDocument();
 
-    //  expect live title to be present
-    screen.getByRole('heading', { name: 'live title' });
+    //  expect live title to be hidden
+    expect(screen.getByText('live title')).not.toBeVisible();
```

Comment:
If you still want to use `getByRole` you can use the `hidden` option: https://testing-library.com/docs/queries/byrole#hidden

Reply:
ok, thanks